### DISCUSSION TITLE
Compatibility with 4-Wire Doorbell / Intercoms

### DISCUSSION THREAD

**Discussion by didgeriwho** [PricelessToolkit/ESPBell-MAX#1] [General]
Hello - I have a 4-wire intercom almost identical to the referenced in this article :
https://chris-m-whong.medium.com/connecting-an-apartment-door-buzzer-to-a-smarthome-hub-4664cf6a3ce4

This intercom doesn't seem to exactly match the 4+N (5-wire) intercoms you've mentioned but I'm curious your thoughts on if the ESPBell-MAX would work for doorbell alerts only (not looking for any door opening functionality). I currently have a Shelly Uni connected to my unit to listen detect signal on the "X" and "1" wire but it requires a separate 12v power source. I'm looking for a more elegant solution so I don't have to have anything plugged in.

![image](https://github.com/PricelessToolkit/ESPBell-MAX/assets/1590874/b2a1fed7-af1f-485b-825e-fa62f4842b81)

**Comment by PricelessToolkit** [comment]
99% sure it will work if between "X and 1" you can detect 24 volts when the doorbell button is pressed
